Source/AliveLibAE/QuikSave.cpp

```diff
@@ -342,6 +342,10 @@ EXPORT void CC Quicksave_LoadFromMemory_4C95A0(Quicksave* quicksaveData)
         0,
         1);
     gMap_5C3030.field_8_force_load = 1;
+
+#if ORIGINAL_GAME_FIXES
+    sIsMudStandingUp_5C3018 = 0;
```

Comment:
should this not actually be part of the quick save data structure? it seems like something that should be persisted?

Reply:
Ideally, probably yes, it didn't seem to have any negative side effects to do it this way during my tests, and I wasn't sure if I could modify quicksave/quickload data without messing with actual save files as well, as it seemed to read the quicksave data from the save file if I'm not mistaken